tests/kafkatest/tests/streams/streams_broker_bounce_test.py

```diff
@@ -141,6 +151,12 @@ def setup_system(self, start_processor=True):
 
         self.kafka = KafkaService(self.test_context, num_nodes=self.replication, zk=self.zk, topics=self.topics)
         self.kafka.start()
+
+        # allow some time for topics to be created
+        wait_until(lambda: self.get_topics_count() >= (len(self.topics) * self.num_kafka_nodes),
```

Comment:
I saw this pattern in `StreamsUpgradeTest` as well, but I'd agree with @vvcephei @mjsax that instead of listing on each node, and check that `sum(num_topics) > num_topics * num_brokers` could we check that `for each broker, list_topics() contains the required topic strings`?

Reply:
And replace this pattern on `StreamsUpgradeTest` as well.